sql/core/benchmarks/ExternalAppendOnlyUnsafeRowArrayBenchmark-results.txt

```diff
@@ -42,4 +42,8 @@ Spilling with 10000 rows:                 Best Time(ms)   Avg Time(ms)   Stdev(m
 UnsafeExternalSorter                                 11             11           1         14.7          68.0       1.0X
 ExternalAppendOnlyUnsafeRowArray                      9             10           1         17.1          58.5       1.2X
 
-
+OpenJDK 64-Bit Server VM 1.8.0_252-8u252-b09-1~16.04-b09 on Linux 4.4.0-178-generic
```

Comment:
You mean the other numbers didn't change at all?

Reply:
Also, could you update the jdk11 one, too: https://github.com/apache/spark/blob/master/sql/core/benchmarks/ExternalAppendOnlyUnsafeRowArrayBenchmark-jdk11-results.txt

Reply:
> You mean the other numbers didn't change at all?

My understanding was to add new benchmark results to the file. I didn't change other results in the files. Do you want me to update all results?

Reply:
One more thing, we will not see performance improvements for the existing benchmarks if the benchmark iterates  over the spilled data. Then comparison will be related to the machine on which benchmark was running and disk speed.

Reply:
> My understanding was to add new benchmark results to the file. I didn't change other results in the files. Do you want me to update all results?

We need to check no perf regression on jdk8/jdk11

Reply:
Yes, I did comparison of the existing micro-benchmark results when patch is applied and when there is no patch.
I do not see regression in the performance for the existing micro-benchmarks. I will push micro-benchmark results for jdk11 soon.
Thank you for help 😊